package.json

```diff
@@ -355,6 +355,7 @@
         "react-modal": "^3.8.1",
         "react-popper": "^1.3.4",
         "react-process-string": "^1.2.0",
+        "react-responsive": "^9.0.0-beta.5",
```

Comment:
Is it possible to use the non-beta version of the library?

Reply:
The beta version seems to be just a js to typescript migration. However there seems to be a memory leak fix included in one of the beta versions. 

https://github.com/yocontra/react-responsive/commit/95b5384b19ef22d375edc5eb63347d5a452e1829

I bumping it down to 8.2.0 and verifying if this is reproducible